README.md

```diff
@@ -49,3 +54,12 @@ otherwise we use the complete URI:
 >>> r.content.decode("utf-8")
 'Sent payload with 27 bytes'
 ```
+
+## Micropython Installation Guide
+
+To be extended:
+
+The dummy libraries `__future__.py`, `abc.py`, `typing.py`, \
+the https://github.com/alexmrqt/micropython-cbor/ CBOR library (specifically the `cbor.py` module/file) \ 
+and `urequests`: `mpremote mip install urequests`, \  
+as well as `datetime`: `mpremote mip install datetime` are needed.
```

Comment:
```suggestion
The dummy libraries `__future__.py`, `abc.py`, `typing.py`, the [micropython-cbor](https://github.com/alexmrqt/micropython-cbor/) library (specifically the `cbor.py` module/file) and `urequests` as well as `datetime` are needed:

$ mpremote mip install urequests
$ mpremote mip install datetime
```

Some minor clean up. Please wrap the last two lines with '```shell' and appropriate ending tag, since doing this in the above Review suggestion will break the suggestion.